src/Altinn.AccessManagement.Core/Enums/ResourceRegistry/ResourceType.cs

```diff
@@ -1,20 +1,19 @@
-﻿namespace Altinn.AccessManagement.Core.Models.ResourceRegistry
+﻿using NpgsqlTypes;
+
+namespace Altinn.AccessManagement.Core.Models.ResourceRegistry
 {
     /// <summary>
     /// Enum representation of the different types of resources supported by the resource registry
     /// </summary>
     public enum ResourceType
     {
+        [PgName("default")]
         Default = 0,
-        
-        Systemresource = 1, 
-        
-        Altinn2 = 2,
-        
-        Altinn3 = 3,
-        
-        MaskinportenSchema = 4,
 
-        Api = 5
+        [PgName("systemresource")]
+        Systemresource = 1,
```

Comment:
Bittelitt usikker på denne. Trenger vi å spesialbehandle denne ressursen på noen måte?

Reply:
Perhaps not, but what should the resource type be if not SystemResource?
We haven't started implementation for it yet, so we could just remove it for now, like the other temp types created during the POC.

Reply:
Can be revisited when we introduce the system resource for Maskinporten schema delegation